robottelo/ui/locators.py

```diff
@@ -1383,6 +1383,15 @@ def __iter__(self):
     "repo.fetch_upstream": (
         By.XPATH, ("//form[@bst-edit-text='repository.docker_upstream_name']"
                    "/div[@class='bst-edit']/div/span[2]")),
+    "repo.fetch_packages": (
+        By.XPATH, "//td[span[text()='Packages']]/following-sibling::td",
```

Comment:
Don't like the idea to rely on localized strings, but those fields have no unique ids, no classes, nothing intresting that we could use. The only other option is to rely on indexes which is even worse, as for different types of packages we have different amount of fields.